### DISCUSSION TITLE
[@types/node] missing parseArgs `allowNegative` option

### DISCUSSION THREAD

**Discussion by ljharb** [DefinitelyTyped/DefinitelyTyped#70056] [Issues with a @types package]
This feature was added in https://github.com/nodejs/node/pull/53107 and released in node v22.4, and the node types seem to stop at v20.

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
Will be added in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/70262

**Reply by Semigradsky** [reply]
Done
